play_field.py

```diff
@@ -2,14 +2,19 @@
 
 
 class PlayField(tk.Frame):
-    def __init__(self, rows, columns):
+    def __init__(self):
         super().__init__()
-        self.rows = rows
-        self.columns = columns
-        self.generate_field()
+        self.buttons = []
 
-    def generate_field(self):
-        for row in range(self.rows):
-            for column in range(self.columns):
+    def generate_field(self, rows, columns):
+        for row in range(rows):
```

Comment:
Чи не треба нам почистити список перед тим, як додавати нові кнопки?
```suggestion
    def generate_field(self, rows, columns):
        self.buttons = []
        for row in range(rows):
```